website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -0,0 +1,12 @@
+## How to use this component
+
+The basic invocation requires `@value` to be passed.
+
+<!-- Getting errors...
+```handlebars
+<Hds::CodeBlock
+  @value="let codeLang='JavaScript';
+    console.log(`I am ${codeLang} code`);"
+/>
+```
```

Comment:
I was getting Syntax errors with rendering the template when attempting to add the `CodeBlock` in the above example.

If I simplified the value of the CodeBlock to just a simple string, then I got an error saying it couldn't find prismjs.

<img width="558" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/42add868-6755-4be2-965e-e11d58a62836">


<img width="558" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/da44b1aa-b80a-4e86-8471-436d20079286">

Reply:
I suspect this is because of the single quotes – I don't think we have a workaround to allow them inside showcase website codeblocks.

The following would work well
```handlebars
<Hds::CodeBlock
  @value="let codeLang=`JavaScript`;
console.log(`I am ${codeLang} code`);"
/>
```

<img width="762" alt="Screenshot 2023-10-05 at 17 06 15" src="https://github.com/hashicorp/design-system/assets/788096/d2c432b8-9712-450a-9af4-9902cbf31087">

We can look more in dept and see if there's a way to allow them.

This raises an interesting challenge, as we already use ember-prism (with an updated dracula theme) for showcasing code snippets in the website, so it's likely to clash in terms of styling with the new component – we shall see.

Reply:
Ok, thanks. As mentioned above even when I simplified the value to remove single quotes, etc I got a different error saying it couldn't find prismjs.

Reply:
I pushed up an updated commented out example.

Reply:
I can't reproduce the prism error. I would run yarn on the `website` (or at the repo root/workspaces level) to make sure the dependency is added.

Reply:
Thanks. I'll try running yarn again. Maybe I was running it just on the component branch mistakenly.

Reply:
That turned out to be it. I had tried running yarn but was running it in the wrong place. Thanks!